config/initializers/locales.rb

```diff
@@ -1,5 +1,5 @@
 SUPPORTED_LOCALES = [:ar, :be, :bg, :bn, :ca, :cs, :da, :de, :el, :en, :es, :et, :fa, :fi, :fr, :he, :hi, :hr, :ht,
                      :hu, :hy, :id, :it, :ja, :ka, :km, :ko, :lt, :lv, :mk, :nl, :pl, :ps, :pt, :ro, :ru, :sk, :so,
                      :sq, :sr, :sw, :th, :tr, :uk, :ur, :uz, :vi, :zh]
-LANGUAGE_ANALYZER_LOCALES = [:es, :en, :de, :fi, :fr, :hu, :it, :ja, :ko, :pt, :ru, :sv, :zh]
+LANGUAGE_ANALYZER_LOCALES = [:bn, :de, :en, :es, :fi, :fr, :hi, :hu, :it, :ja, :ko, :pt, :ru, :sv, :zh]
```

Comment:
Only `:hi` and `:bn` were added. The other changes on this line are just from alphabetization.